### ISSUE TITLE
mcrypt deprecated and removed from php 7.2

### ISSUE THREAD

**Issue by ghost** [ioncube/php-openssl-cryptor#11]
An FYI that as of PHP 7.1 mycrypt is deprecated and removed entirely from version 7.2.

**Comment by nickion** [comment]
Indeed, making this non-mcrypt openssl based class and example all the more relevant! Using openssl over mcrypt has been recommended for several years now.

**Comment by ghost** [comment]
Thanks!  I actually found it and integrated it into my old script.
